src/testHelpers.spec.ts

```diff
@@ -57,10 +57,10 @@ export function expectZeroDiagnostics(arg: { getDiagnostics(): Array<Diagnostic>
     let diagnostics: BsDiagnostic[];
     if (Array.isArray(arg)) {
         diagnostics = arg as BsDiagnostic[];
-    } else if ((arg as any).diagnostics) {
-        diagnostics = (arg as any).diagnostics;
     } else if ((arg as any).getDiagnostics) {
         diagnostics = (arg as any).getDiagnostics();
+    } else if ((arg as any).diagnostics) {
```

Comment:
This is just a test helper function. The `Program` object has `diagnostics` that are program-specific, but also has a `getDiagnostics()` function that rolls up its own diagnostics along with all scopes and files. So we should check for `getDiagnostics()` first since that one will be more complete.